.github/workflows/php.yaml

```diff
@@ -0,0 +1,107 @@
+name: PHP
+
+on:
+    pull_request:
+        branches:
+            - '3.5'
```

Comment:
Prevent double runs for PR-s.

Reply:
Do i need to change it when i add new "main" branch?

Reply:
Yes.

Reply:
What about bugfixes that will target `3.4` branch?

Reply:
But this CI configuration will not run on old branches.

Reply:
So, it will not run for `v3.3`?

Reply:
No, only on the branches listed above.
What branches should I add?

Reply:
> But this CI configuration will not run on old branches.

I've meant it will fail because of Symfony versions.

Reply:
> No, only on the branches listed above. What branches should I add?

Please add `3.5` and `3.6`, hold on with `rebase` until I release `v3.5`.

Reply:
Done 🍏

Reply:
https://github.com/crunzphp/crunz/releases/tag/v3.5.0 released, so You can `rebase` :)

Reply:
Done.